mappings/net/minecraft/block/Block.mapping

```diff
@@ -77,6 +84,8 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 	FIELD field_19061 SOLID_SMALL_SQUARE_SHAPE Lnet/minecraft/class_265;
 	FIELD field_19312 FULL_CUBE_SHAPE_CACHE Lcom/google/common/cache/LoadingCache;
 	FIELD field_20720 opaque Z
+	FIELD field_21207 velocityMultiplier F
+	FIELD field_21208 jumpVelocityMultiplier F
```

Comment:
What blocks override this? If it's slime and honey, I think it should be "bounce velocity multiplier/ratio", since it affects bounces off of the block, not jumps. It could also be called "[restitution](https://en.m.wikipedia.org/wiki/Coefficient_of_restitution)".

Reply:
jump one is not bounce. it's what prevents you from jumping.

Reply:
@Runemoro To clarify, this has nothing to do with the bouncing factors.